crates/opencascade-sys/build.rs

```diff
@@ -36,6 +37,8 @@ fn main() {
     cxx_build::bridge("src/lib.rs")
         .cpp(true)
         .flag_if_supported("-std=c++11")
+        .define("_USE_MATH_DEFINES", "TRUE")
```

Comment:
Can you explain why this line is necessary?

Reply:
Not 100% sure. When cross compiling some math symbols don't get defined unless the flag is set.

Reply:
I see, here's a reference:

https://learn.microsoft.com/en-us/cpp/c-runtime-library/math-constants?view=msvc-170